packages/arb-token-bridge-ui/src/pages/_app.tsx

```diff
@@ -51,6 +52,27 @@ Sentry.init({
   }
 })
 
+if (
+  typeof window !== 'undefined' &&
+  typeof process.env.NEXT_PUBLIC_POSTHOG_KEY === 'string'
+) {
+  posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY, {
+    api_host: 'https://app.posthog.com',
+    loaded: posthog => {
+      if (process.env.NODE_ENV !== 'production') {
+        // when in dev, you can see data that would be sent in prod (in devtools)
+        posthog.debug()
+      }
+    },
+    // store data in temporary memory that expires with each session
+    persistence: 'memory',
+    // by default posthog autocaptures (sends) events such as onClick, etc
+    // we set up our own events instead
+    autocapture: false,
+    disable_session_recording: true
+  })
+}
+
 export default function App({ Component, pageProps }: AppProps) {
   return (
     <>
```

Comment:
why don't we use the `PostHogProvider`?
https://posthog.com/docs/libraries/next-js
https://posthog.com/docs/libraries/react

Reply:
It's because we only capture page views here and it's automatic. Will use `PostHogProvider` when all the events are implemented.